docs/guide/basic-features/connection_type.md

```diff
@@ -36,32 +50,59 @@ When the connection is ready to be returned after used, proceed as follows:
 2. Check whether the idle connection number exceeds  `MaxIdleGlobal`, and if yes, close it directly
 3. Check whether free space remained in the ring of the target connection pool, if yes, put it into the pool, otherwise close it directly
 
-## Parameter Setting
+### Parameter Setting Suggestion
 
 The setting of parameters is suggested as follows:
 - `MaxIdlePerAddress`: the minimum value is 1, otherwise long connections would degenerate to short connections
   - What value should be set should be determined according to the throughput of downstream address. The approximate estimation formula is: `MaxIdlePerAddress = qps_per_dest_host*avg_response_time_sec`
-  - For example, the cost of each request is 100ms, and the request spread to each downstream address is 100QPS, the value is suggested to set to 10, because each connection handles 10 requests per second, 100QPS requires 10 connections to handled
+  - For example, the cost of each request is 100ms, and the request spread to each downstream address is 100QPS, the value is suggested to set to 10, because each connection handles 10 requests per second, 100QPS requires 10 connections to handle
   - In the actual scenario, the fluctuation of traffic is also necessary to be considered. Pay attention, the connection within MaxIdleTimeout will be recycled if it is not used
-  - Summary: this value be set too large or too small would lead to degenerating to short connection
+  - Summary: this value be set too large or too small would lead to degenerating to the short connection
 - `MaxIdleGlobal`: should be larger than the total number of `downstream targets number * MaxIdlePerAddress`
   - Notice: this value is not very valuable, it is suggested to set it to a super large value. In subsequent versions, considers discarding this parameter and providing a new interface
 - `MaxIdleTimeout`: since the server will clean up inactive connections within 10min, the client also needs to clean up long-idle connections in time to avoid using invalid connections. This value cannot exceed 10min when the downstream is also a Kitex service
 
+## Connection Multiplexing
+
+The client invokes the Server only need one connection normally when enabling Connection Multiplexing. Connection Multiplexing not only reduces the number of connections but also performs better than Connection Pool.
+
+Special Note:
+
+1. Connection Multiplexing here is just for Thrift and Kitex Protobuf protocol. If you choose the gRPC protocol, it is Connection Multiplexing mode.
```

Comment:
If you choose the gRPC protocol, it utilizes Connection Multiplexing mode by default.